Management/src/ActuatorWeb/README.md

```diff
@@ -0,0 +1,201 @@
+
+ActuatorWeb and ActuatorAPI form an ASP.NET Core-powered sample application that demonstrates how to use several
+Steeltoe libraries on their own and with additional tools such
+as [Tanzu Apps Manager](https://techdocs.broadcom.com/us/en/vmware-tanzu/platform/tanzu-platform-for-cloud-foundry/10-0/tpcf/console-index.html)
+and [Spring Boot Admin](https://docs.spring-boot-admin.com/) used for managing and monitoring applications.
+
+These samples also illustrate how to have application metrics captured and exported to
+the [Metrics Registrar](https://techdocs.broadcom.com/us/en/vmware-tanzu/platform/tanzu-platform-for-cloud-foundry/10-0/tpcf/metric-registrar-index.html)
+service so that application metrics can be viewed in any tool that is able to consume those metrics from
+the [Cloud Foundry Loggregator](https://github.com/cloudfoundry/loggregator-release).
+Several tools exist that can do this,
+including [App Metrics for VMware Tanzu](https://techdocs.broadcom.com/us/en/vmware-tanzu/platform-services/app-metrics-for-tanzu/2-2/app-metrics/index.html).
+
+> [!NOTE]  
+> While this application is intended to be used with [ActuatorApi](../ActuatorApi/), it _can_ be used on its own if you
+> are looking for a simpler actuator or metrics example.
+
+## General pre-requisites
+
+1. Installed .NET 8 SDK
+1. Optional:
+    * [VMware Tanzu Platform for Cloud Foundry](https://techdocs.broadcom.com/us/en/vmware-tanzu/platform/tanzu-platform-for-cloud-foundry/10-0/tpcf/concepts-overview.html)
+    * [Cloud Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html)
+    * [Metrics Registrar for VMware Tanzu Application Service](https://techdocs.broadcom.com/us/en/vmware-tanzu/platform/tanzu-platform-for-cloud-foundry/10-0/tpcf/metric-registrar-index.html)
+    * [App Metrics for VMware Tanzu](https://techdocs.broadcom.com/us/en/vmware-tanzu/platform/tanzu-platform-for-cloud-foundry/10-0/tpcf/metric-registrar-index.html)
+    * [VMware MySQL for Tanzu Application Service](https://techdocs.broadcom.com/us/en/vmware-tanzu/data-solutions/tanzu-for-mysql-on-cloud-foundry/3-3/mysql-for-tpcf/about_mysql_vms.html)
+    or a [VMware Tanzu Cloud Service Broker](https://techdocs.broadcom.com/us/en/vmware-tanzu/platform-services.html)
+
+## Running locally
+
+This application has a number of moving pieces, some of which can be used in multiple ways. In order to experience all
+of the functionality in a local environment, you will need to meet additional pre-requisites:
+
+* Installed container orchestrator (such as [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+  or [Podman Desktop](https://podman-desktop.io/))
+* Optional: an IDE (such as Visual Studio), for performing the requests defined
+  in [ActuatorWeb.http](./ActuatorWeb.http) and [ActuatorApi.http](../ActuatorApi/ActuatorApi.http).
+
+> [!NOTE]  
+> Use the [docker-compose file](../docker-compose.yaml) to run all of the backing services used by these samples.
+>
+> ```bash
```

Comment:
Other places use `shell`, what's the difference?

Reply:
The linter started complaining that it doesn't recognize shell... idk _why_ because I'm pretty sure I've been using the same linter for like 5 years at this point

Reply:
So we've settled on using `shell` instead of `bash`? Please open an issue to align other places within samples and docs.

Reply:
I think the verdict was that they're ultimately rendered the same, and that the linter is being silly

https://github.com/SteeltoeOSS/Samples/issues/351